addons/test_mail/tests/test_mail_followers.py

```diff
@@ -986,9 +986,14 @@ def setUpClass(cls):
         super().setUpClass()
         cls.user_portal = cls._create_portal_user()
         cls.partner_portal = cls.user_portal.partner_id
-        cls.test_record = cls.env['mail.test.simple'].with_context(cls._test_context).create({'name': 'Test'})
-        cls.test_record_unfollow = cls.env['mail.test.simple.unfollow'].with_context(cls._test_context).create(
-            {'name': 'unfollow'})
+        cls.test_record = cls.env['mail.test.simple'].with_context(
+            cls._test_context,
+            email_notification_show_header=True,
+            email_notification_enable_footer=True).create({'name': 'Test'})
+        cls.test_record_unfollow = cls.env['mail.test.simple.unfollow'].with_context(
+            cls._test_context,
+            email_notification_show_header=True,
+            email_notification_enable_footer=True).create({'name': 'unfollow'})
```

Comment:
Can't you put those context keys in message_post call instead ? Would be clearer to me than at creation time :)